packaging/functions.sh

```diff
@@ -40,7 +41,7 @@ install_assemblies() {
 		rm -rf "${SRC_PATH}/OpenRA."*/obj
 		rm -rf "${SRC_PATH:?}/bin"
 
-		msbuild -verbosity:m -nologo -t:Build -restore -p:Configuration=Release -p:TargetPlatform="${TARGETPLATFORM}" -p:Mono=true
+		$MSBUILD -verbosity:m -nologo -t:Build -restore -p:Configuration=Release -p:TargetPlatform="${TARGETPLATFORM}" -p:Mono=true
```

Comment:
Do you happen to know if what `dotnet msbuild` produces is compatible with Mono?

Reply:
the dotnet msbuild results in a working OpenRA package  / is compatible with Mono.

Reply:
the dotnet msbuild results in a working OpenRA package  / is compatible with Mono.

Reply:
Can you explain the motivation for this?

This code path is supposed to be for cases where we explicitly don't want to be using `dotnet`.

Reply:
The build fails for me without calling dotnet.

Reply:
The build fails for me without calling dotnet.

Reply:
This code path should only be running if you are explicitly running `make install RUNTIME=mono`. What happens if you just do `make install`?

Reply:
OpenRA requires the 'msbuild -verbosity:m -nologo' tool provided by Mono >= 5.18.

however, I have mono-6.12.0.122 installed and it works. But I guess it no longer has msbuild? So I looked into building a standalone msbuild but it brought me to dotnet anyway... so I decided to just use the dotnet version installed, since the mono project was redirecting to dotnet anyway.

Reply:
That error message suggests that you are trying to compile the `release-20210321` tag. The toolchain has changed significantly since then! Can you try again on bleed?

Reply:
I see. I hadn't tested building without RUNTIME=mono on bleed. So this is a non issue. The unrelated odd issue I have on bleed tho is for whatever reason, is this:

>Exception of type 'System.TimeoutException' was thrown.
  Failed to download package 'System.Runtime.Extensions.4.3.0' from 'https://api.nuget.org/v3-flatcontainer/system.runtime.extensions/4.3.0/system.runtime.extensions.4.3.0.nupkg'.
  The download of 'https://api.nuget.org/v3-flatcontainer/system.runtime.extensions/4.3.0/system.runtime.extensions.4.3.0.nupkg' timed out because no data was received for 60000ms.
    Exception of type 'System.TimeoutException' was thrown.